app/views/participants/fields/_financial.html.erb

```diff
@@ -5,7 +5,11 @@
 		<%= f.input :free_reduced_lunch_signed_up, :label => "Signed up" %>
 	<% end %>
 
-	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household", :hint => "Include the student in this count." %>
+	<% if !(current_user.type = 'Participant') %>
```

Comment:
This `if` statement could be folded into the `:hint` value field, because it looks like that's the only difference here between the two lines